src/main/java/no/sikt/sws/WorkspaceResponse.java

```diff
@@ -0,0 +1,22 @@
+package no.sikt.sws;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.databind.JsonNode;
+
+public class WorkspaceResponse {
+
+    @JsonProperty("account_identifier")
```

Comment:
should we use _ or - ?

Reply:
https://opensource.zalando.com/restful-api-guidelines/#118

snake_case is correct :)